reproduciblebuilds/README.md

```diff
@@ -0,0 +1,65 @@
+# Bootstrappable Cryptopower Builds
+
+This directory contains the files necessary to perform bootstrappable cryptopower builds.
+
+Bootstrappability furthers our binary security guarantees by allowing us to audit and reproduce our toolchain instead of blindly trusting binary downloads.
+
+We achieve bootstrappability by using Docker.
+
+## Usage
+
+Note: You need to have [Docker](https://www.docker.com/) to build.
+
+in the cryptopower root directory and execute:
+
+`./reproducible_builds.sh {target}`
+
+select your target from [Build Targets](https://github.com/crypto-power/cryptopower/tree/master/reproduciblebuilds#build-targets)
+
+e.g `./reproducible_builds.sh linux-amd64-binary`
+
+### Verification
+
+You can then verify if the binaries are reproducible using a tool like [diffoscope](https://diffoscope.org/) or [reprotest](https://salsa.debian.org/reproducible-builds/reprotest)
+
+#### Using Diffoscope
+
+e.g `diffoscope binary1 binary2`
```

Comment:
Shouldn't you specify a standard reference binary the users can use to compare against?

Reply:
it's not necessary, reproducible build isn't a beginner or novice task, choosing to do it means you have an understanding of what you are trying to achieve which is, comparing a release binary to one you've replicated using docker. those binaries would be available on the release page, maybe i can add a link to the release page just for ease, and also specify where the reproduced binary can be found